css/style.css

```diff
@@ -237,6 +237,22 @@ canvas {
     grid-column: 1 / 3;
     grid-row: 3;
 }
+.output {
+    border: 1px inset #999;
+    padding: 2px;
+    width: calc(100% - 6px);
+    height: calc(100% - 6px);
+    overflow: auto;
+}
+.output .info {
+    color: green;

```

Comment:
Perhaps this should be gray instead? Since it feels to me more like “background” / automatic information.